nnfabrik/main.py

```diff
@@ -69,24 +120,29 @@ def add_entry(
         self,
         model_fn,
         model_config,
-        model_fabrikant=None,
         model_comment="",
+        model_fabrikant=None,
         skip_duplicates=False,
+        return_key_only=True,
```

Comment:
It's missing documentation, but as far as I understand if `return_key_only=True` it only return the primary keys?

Reply:
Also I noticed the explanation for what it returns is different from that of Dataset table, although it's returning the same things (but for model).

Reply:
Yeah updated to make them the same now

Reply:
Maybe change to `return_primary_only` or `return_pkey_only`? `key` is kind of confusing, I'd say.